x-pack/plugins/fleet/server/routes/package_policy/handlers.ts

```diff
@@ -79,18 +79,20 @@ export const createPackagePolicyHandler: RequestHandler<
 > = async (context, request, response) => {
   const soClient = context.core.savedObjects.client;
   const esClient = context.core.elasticsearch.client.asCurrentUser;
-  const user = (await appContextService.getSecurity()?.authc.getCurrentUser(request)) || undefined;
+  const user = appContextService.getSecurity()?.authc.getCurrentUser(request) || undefined;
```

Comment:
saw this in VS code:
<img width="612" alt="Screen Shot 2021-04-11 at 3 50 19 PM" src="https://user-images.githubusercontent.com/57655/114319028-ac0e4b80-9add-11eb-8811-be0153b5868a.png">

Everything I found also indicates `getCurrentUser` is sync https://github.com/elastic/kibana/blob/6ecffcc57c9a5b0328257cd25b737140a8471dda/x-pack/plugins/security/server/authentication/authentication_service.ts#L59 & https://github.com/elastic/kibana/blob/6ecffcc57c9a5b0328257cd25b737140a8471dda/x-pack/plugins/security/server/authentication/authentication_service.ts#L161-L162